[AOTI] Print out error msg when nvcc compiler fails

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #157203

Summary: To debug https://github.com/pytorch/pytorch/issues/156930. Not able to reproduce the problem locally.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov

desertfire committed:
[AOTI] Print out error msg when nvcc compiler fails

Summary: To debug https://github.com/pytorch/pytorch/issues/156930. Not able to reproduce the problem locally.

[ghstack-poisoned]

jansel commented on torch/_inductor/codecache.py:
```diff
@@ -2143,9 +2143,15 @@ def _pad_to_alignment(raw_bytes: bytes) -> bytes:
+                            check=True,
+                        )
+                    except subprocess.CalledProcessError as e:
+                        print(f"{cmd} failed with: {e.stderr}")
```
```suggestion
                        print(f"{cmd} failed with:\nstdout:\n{e.stdout}\nstderr:\n{e.stderr}", file=sys.stderr)
                        raise
```

I assume you want to re-raise here?

desertfire committed:
Update torch/_inductor/codecache.py

Co-authored-by: Jason Ansel <jansel@meta.com>

desertfire committed:
Update on "[AOTI] Print out error msg when nvcc compiler fails"


Summary: To debug https://github.com/pytorch/pytorch/issues/156930. Not able to reproduce the problem locally.

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Print out error msg when nvcc compiler fails"


Summary: To debug https://github.com/pytorch/pytorch/issues/156930. Not able to reproduce the problem locally.

[ghstack-poisoned]
